dashboard/admin.py

```diff
@@ -11,6 +13,13 @@ class CourseViewOptionInline(admin.StackedInline):
 
 class CourseAdmin(admin.ModelAdmin):
     inlines = [CourseViewOptionInline,]
+    list_display = ('name', 'term_id','_courseviewoption')
```

Comment:
For human readabilities:

name -> course name or user name
_courseviewoption -> course view option

Reply:
do not see list_display or list_select_realted used anywhere....

Reply:
These are attributes of ModelAdmin (which this is extending)
https://docs.djangoproject.com/en/1.11/ref/contrib/admin/#django.contrib.admin.ModelAdmin.list_display

They are used to define which columns appear in the table. It's pretty neat. 

These fields have to either map against the columns in the database or a method.

The verbose_name from the model is used in the display, except for this special new method I added to display the courseviewoptions table. I had to add this special method here so it would display the newlines as html <br> on separate lines for better readability in the table.

Reply:
Nice!